doc/man/8/ceph-disk.rst

```diff
@@ -264,6 +337,22 @@ Options
 
    Provide init system to manage the OSD directory.
 
```

Comment:
The lines below can be removed. And the --mark-init and --activate-key above should also be removed as they are not options that are common to all subcommands. But that does not need to be in this pull request ;-)